aiohttp_cache/backends.py

```diff
@@ -91,49 +91,23 @@ def __init__(self, expiration: int = 300):
 # --------------------------------------------------------------------------
 # REDIS BACKEND
 # --------------------------------------------------------------------------
-class RedisConfig(_Config):
```

Comment:
What is the reason to remove redis config?

Reply:
Redis URL is more convenient.